### PR TITLE
🪟 🚦 Add base E2E test for new stream table (part 1)

### PR DESCRIPTION
**Pull request by dizel852**
## What
Related to https://github.com/airbytehq/airbyte/issues/18267

## How
The test skips user flow: create source and destination connectors via UI.
We create them via API request and then start creating a new connection flow from choosing the existing ones.

Areas covered in the base test:
1. Selecting Source and Destination connectors from existing ones
2. Page routing (at what page we currently are after actions)
3. Changing replication frequency (only "Manual" options)
4. Connection set up - partial check API response(connection name and replication frequency)

### CODE REVIEW

File: airbyte-webapp-e2e-tests/cypress/pages/newConnection.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+type ConnectorType = "source" | "destination";
```
Comment (edmundito) [inline airbyte-webapp-e2e-tests/cypress/pages/newConnection.ts:1]: File should be called `newConnectionPage`

File: airbyte-webapp-e2e-tests/cypress/integration/streamTable.spec.ts [hunk 2]
```diff
@@ -0,0 +1,131 @@
+import { initialSetupCompleted } from "commands/workspaces";
```
Comment (edmundito) [inline airbyte-webapp-e2e-tests/cypress/integration/streamTable.spec.ts:1]: This file should go in `integration/connection/` (we can move the other files later)

File: airbyte-webapp/src/components/connection/ConnectionOnboarding/ConnectionOnboarding.tsx [hunk 3]
```diff
@@ -241,7 +241,7 @@ export const ConnectionOnboarding: React.FC<ConnectionOnboardingProps> = ({ onCr
         </div>
       </div>
       <div className={styles.footer}>
-        <Button onClick={() => onCreate()} size="lg" data-id="new-connection">
+        <Button onClick={() => onCreate()} size="lg" data-id="new-connection" data-testid="new-connection-button">
```
Comment (edmundito) [inline airbyte-webapp/src/components/connection/ConnectionOnboarding/ConnectionOnboarding.tsx:244]: is `data-id` still important? What is it being used for currently?

**Review by edmundito — COMMENTED** [review]
Looks good, just had a couple of nits and a question.

File: airbyte-webapp-e2e-tests/cypress/pages/newConnection.ts (hunk 1 above)
Comment (dizel852) [reply airbyte-webapp-e2e-tests/cypress/pages/newConnection.ts:1]: Renamed

File: airbyte-webapp/src/components/connection/ConnectionOnboarding/ConnectionOnboarding.tsx (hunk 3 above)
Comment (dizel852) [reply airbyte-webapp/src/components/connection/ConnectionOnboarding/ConnectionOnboarding.tsx]: I've double-checked, seems like we don't use it at all, so I removed it.

File: airbyte-webapp-e2e-tests/cypress/integration/streamTable.spec.ts [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+  waitForDiscoverSchemaRequest,
+  waitForGetSourceDefinitionsRequest,
+  waitForGetSourcesListRequest,
+} from "commands/interceptors";
+import { Connection, Destination, Source } from "commands/api/types";
+import { selectSchedule } from "pages/replicationPage";
+import { runDbQuery } from "commands/db/db";
+import { createUsersTableQuery, dropUsersTableQuery } from "commands/db/queries";
+
+// TODO: Disable before merge
```
[the remark this answers is not in the document]
Comment (dizel852) [reply airbyte-webapp-e2e-tests/cypress/integration/streamTable.spec.ts]: oh, probably because of "base" tests 😄

**Review by edmundito — APPROVED** [review]

**Pull request merged**